src/morello-install/README.md

```diff
@@ -640,28 +651,24 @@ If there are further configuration settings to change, select an appropriate
 option from the menu and press Space.
 Otherwise, press Enter to complete the installation:
```

Comment:
```suggestion
Otherwise, press navigate to the Finish button and press Enter to complete the installation:
```

Reply:
I believe Finish is selected by default, and if you want to choose one of the list items you have to also navigate left to Select. It’s a bit confusing and I’m not sure it’s an improvement over the previous interface.

Reply:
OK, but we're not about to change the button now, so, how do we best explain it?

Reply:
See the first part of the message?..

Reply:
Better?

Reply:
Wow, the new UI is horrible